charts/argo-cd/values.yaml

```diff
@@ -1296,16 +1290,37 @@ server:
   #    cpu: 50m
   #    memory: 64Mi
 
-  ## Certificate configuration
+  # TLS certificate configuration via cert-manager
+  ## Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/tls/#tls-certificates-used-by-argocd-server
   certificate:
-    # -- Deploy a Certificate resource (requires cert-manager)
+    # -- Deploy a Certificate resource
     enabled: false
     # -- Certificate primary domain (commonName)
     domain: argocd.example.com
-    # -- The requested 'duration' (i.e. lifetime) of the Certificate. Value must be in units accepted by Go time.ParseDuration
-    duration: ""
-    # -- How long before the currently issued certificate's expiry cert-manager should renew the certificate. Value must be in units accepted by Go time.ParseDuration
-    renewBefore: ""
+    # -- Certificate Subject Alternate Names (SANs)
+    additionalHosts: []
+    # -- The requested 'duration' (i.e. lifetime) of the certificate.
+    # @default -- `2160h` = 90d
+    ## NOTES:
+    ## - Value must be in units accepted by Go time.Duration string format, which does not allow `d` (days) suffix. You must specify values using `s`, `m` and `h` suffixes instead.
+    ## - Minimum value is 1h
+    duration: 2160h
```

Comment:
Why are you "preloading" values which are not necessary? I recommend to leave the defauls as-is, then the defaults are either inherited from the Issuer or cert-manager defaults.

Reply:
These values are actually defaults if you do not specify them. See https://cert-manager.io/docs/usage/certificate/#renewal

Reply:
I'm fine with keeping with empty string. Maybe just node in docs what defaults are?

Reply:
@mkilchhofer changed this to empty strings with links to upstream docs